### DISCUSSION TITLE
DComic v1.4.10-beta

### DISCUSSION THREAD

**Discussion by hanerx** [hanerx/flutter_dmzj#33] [General]
鬼知道我干了啥，我把整个flutter给更新了，然后依赖就炸了。。。然后我把依赖都更新了，然后兼容性炸了。。。然后我把兼容性解决了，我又开始手贱了。。。
我发现嗷，在我后台看到应该有那么一两百的人在用这个玩意的，但是每次修bug我都不知道你们到底是个啥情况，你们也不发你们的错误日志，所以我就干脆自己动手了——这个版本我加上了数据收集也就是Google analysis，不过放心这个只是拿来记录你的错误日志和各个页面的路由的，你的账号啥的都是安全的，数据采用匿名方式上传。以后正式版还会加上关闭的按钮方便一些对隐私有要求的用户，第二个是通过这个功能实现一个app内置的bug报告系统，你可以直接在里面报告bug然后由这玩意整合提交。~~那么看了这么多字我来说明为啥要加这个，真正原因是——That’s cool! 你是不知道这个东西有多酷，**超牛逼！** 各种信息自动归总，哇，超好看，超方便。有一说一我看完这个东西的后台之后我感觉如果不加上这玩意我都对不起自己。~~
以后每个版本都会在GitHub上开一个讨论，欢迎来写点东西 [https://github.com/hanerx/dcomic/discussions](https://github.com/hanerx/dcomic/discussions)
这个版本改动主要围绕被我手贱升级的flutter的兼容性修改，然后加了一些本来没得的功能，修复了一些1.4.9的bug
- 修复了历史记录跳转的bug
- 修改了历史记录的样式，和外面的统一了起来
- 给每个页面加上了路由监控，让我能知道到底你的app在哪里爆炸了和用的比较频繁的页面好做针对性优化
- 添加了本地文件夹形式的漫画导入
- 修复了状态栏是黑色的很丑的bug
- 修改了首页卡的样式，现在首页卡的样式变得更加好看了，代码也从原来的一坨变成了正常人应该写得出来的代码
- 修复了一些显示bug
- 添加了Google crash报告
- 修改了大量依赖项，让依赖能够在新flutter上运行
- 修改了manga解压的插件，应该能更加安全的解压大文件了（好像并没有卵用，只是从崩溃变成了会报错）
- 修改了首页的进入漫画的方式，会自动切换漫画源而不是记录你上次的漫画源，提供更好的体验

<hr /><em>This discussion was created from the release <a href='https://github.com/hanerx/dcomic/releases/tag/v1.4.10-beta.1'>DComic v1.4.10-beta.1</a>.</em>

**Comment by ChellyL** [comment]
请问大佬有无兴趣加一个拷贝漫画（ https://www.copymanga.com/）的源(‾◡◝)

**Reply by hanerx** [reply]
可以整一个，建议你开个issue我好整理
